content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -39,22 +39,22 @@ kubectl:
 kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
 ```
 ```
-deployment "nginx-app" created
+pod/nginx-app created
 ```
 
 {{< note >}}
-`kubectl` commands print the type and name of the resource created or mutated, which can then be used in subsequent commands. You can expose a new Service after a Deployment is created.
+`kubectl` commands print the type and name of the resource created or mutated, which can then be used in subsequent commands. You can expose a new Service after a Pod is created.
 {{< /note >}}
 
 ```shell
 # expose a port through with a service
-kubectl expose deployment nginx-app --port=80 --name=nginx-http
+kubectl expose pod nginx-app --port=80 --name=nginx-http
 ```
 ```
 service "nginx-http" exposed
 ```
 
-By using kubectl, you can create a [Deployment](/docs/concepts/workloads/controllers/deployment/) to ensure that N pods are running nginx, where N is the number of replicas stated in the spec and defaults to 1. You can also create a [service](/docs/concepts/services-networking/service/) with a selector that matches the pod labels. For more information, see [Use a Service to Access an Application in a Cluster](/docs/tasks/access-application-cluster/service-access-application-cluster).
+By using kubectl, you can create a [Pod](/docs/concepts/workloads/pods/pod/) to ensure that pod are running nginx. You can also create a [service](/docs/concepts/services-networking/service/) with a selector that matches the pod labels. For more information, see [Use a Service to Access an Application in a Cluster](/docs/tasks/access-application-cluster/service-access-application-cluster).
```

Comment:
> By using kubectl, you can create a [Pod](/docs/concepts/workloads/pods/pod/) to ensure that pod are running nginx.

The control plane _doesn't_ provide an ongoing commitment to keep the Pod running.

Reply:
ok，delete it ,and add notes tells users how to ensure pod is available in the whole cluster.